travis/deploy-docker.sh

```diff
@@ -6,6 +6,7 @@ if [[ -n "$DOCKER_PASS" && -n "$DOCKER_USER" && -n "$DOCKER_REGISTRY" ]]; then
 
 	# log in to Docker registry
 	echo "$DOCKER_PASS" | docker login --password-stdin --username "$DOCKER_USER" "$DOCKER_REGISTRY"
+	echo "$DOCKER_PASS_MIRROR" | docker login --password-stdin --username "$DOCKER_USER_MIRROR" "$DOCKER_REGISTRY_MIRROR"
```

Comment:
Can you add something like this?
```
if [ -n "$DOCKER_USER_MIRROR" -a -n "$DOCKER_PASS_MIRROR" -a -n "$DOCKER_REGISTRY_MIRROR" ]; then
```

I'm thinking of the scenario where someone forks our github repo and would only need to tweak their .travis.yml accordingly to work with their deploy repos (and maybe they don't have or want a mirror).